tools/ci/run_tc.py

```diff
@@ -146,25 +146,20 @@ def start_dbus():
     # https://source.chromium.org/chromium/chromium/src/+/main:content/app/content_main.cc;l=220;drc=0bcc023b8cdbc073aa5c48db373810db3f765c87.
     os.environ["DBUS_SESSION_BUS_ADDRESS"] = "autolaunch:"
 
-
 def install_chrome(channel):
-    if channel in ("experimental", "dev"):
-        deb_archive = "google-chrome-unstable_current_amd64.deb"
-    elif channel == "beta":
-        deb_archive = "google-chrome-beta_current_amd64.deb"
-    elif channel == "stable":
-        deb_archive = "google-chrome-stable_current_amd64.deb"
-    else:
-        raise ValueError("Unrecognized release channel: %s" % channel)
-
-    dest = os.path.join("/tmp", deb_archive)
-    deb_url = "https://dl.google.com/linux/direct/%s" % deb_archive
-    with open(dest, "wb") as f:
-        get_download_to_descriptor(f, deb_url)
-
-    run(["sudo", "apt-get", "-qqy", "update"])
-    run(["sudo", "gdebi", "-qn", "/tmp/%s" % deb_archive])
-
+    # Interpret experimental channels as Canary.
+    if channel in ("experimental", "nightly", "canary"):
+        channel = "canary"
+    run(["sh", "-c", "./wpt install --channel canary chrome browser"])
+
+    # Chrome for Testing needs additional files in the download directory to run all features.
```

Comment:
I don't really know what this comment is trying to say. The code doesn't seem to end up adding additional files, but moving the downloaded ones?

Reply:
When I was setting this up and running the integration tests, I was moving the `chrome` binary only to the new directory, which was causing unintended side effects and a failure for the browser tests to run. The problem was that the binary relies on (at least some of) the other files downloaded to also exist in the same directory for binary to detect them and run properly.

The comment could definitely be worded better - I'll update it. 😅 Also, if there is a better or more preferable approach to doing this, I'd be glad to update this.